concreteproperties/concrete_section.py

```diff
@@ -1240,6 +1269,12 @@ def micurve(progress=None):
 
         # cut diagram at max_comp
         if max_comp:
+            # check input - if value greater than maximum compression
+            if max_comp > mi_results.results[0].n:
+                msg = f"max_comp={max_comp} is greater than the maximum axial load "
```

Comment:
think this should be axial capacity, not load?

```suggestion
                msg = f"max_comp={max_comp} is greater than the maximum axial capacity "
```

Reply:
Yes good point!